### PR TITLE
Upgraded frontend-build version to v12

### PR DESCRIPTION
**Pull request by BilalQamar95**
**Ticket:**
[42: Upgrade eslint to v8.x](https://github.com/openedx/frontend-wg/issues/42)

**What changed?**
- Updated `frontend-build `to v12 (Eslint was updated in `frontend-build` version resulting in it's version being bumped to v12. This PR updates `frontend-build` to reciprocate eslint version update)
- Resolved `eslint` issues

### CODE REVIEW

**Comment by davidjoy** [comment]
Oof - package-lock.json conflicts.  Mind rebasing?

File: src/account-settings/AccountSettingsPage.jsx [hunk 1]
```diff
@@ -146,28 +146,13 @@ class AccountSettingsPage extends React.Component {
...
   }));
 
-  sortDates = (a, b) => {
-    const aTimeSinceEpoch = new Date(a).getTime();
-    const bTimeSinceEpoch = new Date(b).getTime();
-
-    return bTimeSinceEpoch - aTimeSinceEpoch;
-  }
-
-  sortVerifiedNameRecords = verifiedNameHistory => {
```
Comment (davidjoy) [inline src/account-settings/AccountSettingsPage.jsx:156]: Was this function unused?

**Review by davidjoy — APPROVED** [review]
This looks good - think we just need to resolve the conflict.  (And the sanity check around whether `sortVerifiedNameRecords` was in use)

File: src/account-settings/AccountSettingsPage.jsx (hunk 1 above)
Comment (BilalQamar95) [reply src/account-settings/AccountSettingsPage.jsx:156]: Yes, this function was unused & was triggering `react/no-unused-class-component-methods` rule, therefore I removed it.

**Pull request merged**